### PR TITLE
Retire welcome component

### LINKED ISSUE #3008: Remove /auth/welcome page
### Description
This page can be removed as it's no longer used and replace with landing page. 

### Acceptance criteria
- [ ] Remove all the links to `/auth/welcome` and replace it with landing page
- [ ] Modify the e2e tests that use it to lead to landing page instead and adapt the tests so it keeps working
- [ ] WelcomeComponent can be delete as it should be no longer used

### PR DESCRIPTION
**Pull request by xmano**
Closes #3008

### CODE REVIEW

File: apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-end.spec.ts [hunk 1]
```diff
@@ -18,7 +19,9 @@ const SYNOPSIS = 'An up-close-and-personal portrait of the fashion icon, Karl La
 const PRODUCTION_YEAR = '2006';
 
 beforeEach(() => {
-  clearDataAndPrepareTest();
+  clearDataAndPrepareTest('/');
+  const p1 = new LandingPage();
+  p1.clickSignup();   
   signInAndNavigateToMain();
```
Comment (Hugo-Hontans) [inline apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel-end.spec.ts:25]: Here you can add 
```
  const p1 = new LandingPage();
  p1.clickSignup();   
```
directly inside signInAndNavigateToMain function I think

File: libs/e2e/src/lib/utils/functions.ts [hunk 2]
```diff
@@ -13,8 +13,7 @@ export function clearDataAndPrepareTest(path: string = '/auth') {
 
 /** Start on AuthWelcomePage, on AuthLoginPage and signin. You have to create a new page depending of the app. */
 export function signIn(user: Partial<User>) {
-  const p1: AuthWelcomePage = new AuthWelcomePage();
-  const p2: AuthLoginPage = p1.clickCallToAction();
+  const p2: AuthLoginPage = new AuthLoginPage();
```
Comment (Hugo-Hontans) [inline libs/e2e/src/lib/utils/functions.ts:16]: change p2 for p1 so plz. Keep the same consistency starting with p1

File: apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 3]
```diff
@@ -17,7 +18,9 @@ import {
 describe('User can navigate to the movie tunnel pages start and main.', () => {
   // Log in and create a new movie
   it('Login into an existing account, navigate on titles list page, go to movie tunnel start page, go on movie tunnel main page', () => {
-    clearDataAndPrepareTest();
+    clearDataAndPrepareTest('/');
+    const p1 = new LandingPage();
+    p1.clickSignup();    
     signInAndNavigateToMain();
```
Comment (Hugo-Hontans) [inline apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel.spec.ts:24]: same

File: apps/catalog/catalog-e2e/src/integration/dashboard/contract-tunnel.spec.ts [hunk 4]
```diff
@@ -24,13 +25,15 @@ const TRIGGERING_EVENT = 'First theatrical release';
 
 beforeEach(() => {
   clearDataAndPrepareTest();
+  cy.visit('/');
```
Comment (Hugo-Hontans) [inline apps/catalog/catalog-e2e/src/integration/dashboard/contract-tunnel.spec.ts:28]: By default inside clearDataAndPrepareTest we use `cy.visit('/auth')` and you can add an other path directly. I let you check inside this function.
And if the function goes in route `/auth` you should be directly on AuthLoginPage so you don't need
```
  const p1 = new LandingPage();
  p1.clickSignup();  
```
I think

File: libs/e2e/src/lib/utils/functions.ts (hunk 2 above)
Comment (xmano) [reply libs/e2e/src/lib/utils/functions.ts:16]: OK

File: apps/catalog/catalog-e2e/src/integration/dashboard/contract-tunnel.spec.ts (hunk 4 above)
Comment (xmano) [reply apps/catalog/catalog-e2e/src/integration/dashboard/contract-tunnel.spec.ts:28]: For flexibility I move this inside `signInAndNavigateToMain`

**Pull request merged**